src/core_plugins/kibana/public/visualize/listing/create_or_delete_button.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { CreateButtonLink } from 'ui_framework/components/button/create_button_link';
+import { DeleteButton } from 'ui_framework/components/button/delete_button';
+
+export function CreateOrDeleteButton({ showCreate, doDelete }) {
+  if (showCreate) {
+    return <CreateButtonLink
+      href = "#/visualize/new"
+      tooltip = "Create new visualization"
+    />;
+  } else {
+    return <DeleteButton
+      onClick={() => doDelete() }
+      tooltip="Delete selected visualizations"
+    />;
+  }
+}
+
+CreateOrDeleteButton.propTypes = {
+  showCreate: React.PropTypes.bool,
+  doDelete: React.PropTypes.func
```

Comment:
Can we change this to `isCreate` to indicate that it's a boolean and not a function, and `onDeleteClick` to indicate that it's a callback?